db/data/regions/VN/en.yml

```diff
@@ -8,18 +8,29 @@ en:
             default: Ward
             optional: Ward (optional)
           errors:
-            blank: Enter a ward
-            too_long: Ward is too long (maximum is 255 characters)
-            contains_emojis: Ward cannot contain emojis
-            contains_mathematical_symbols: Ward cannot contain mathematical symbols
-            contains_restricted_characters: Ward can only contain letters, numbers,
+            blank_instructional: Enter a ward
+            blank_informative: Ward is missing
+            too_long_instructional: Ward is too long (maximum is 255 characters)
+            too_long_informative: War is too long
+            contains_emojis_instructional: Ward cannot contain emojis
+            contains_emojis_informative: Ward cannot contain emojis
+            contains_mathematical_symbols_instructional: Ward cannot contain mathematical symbols
+            contains_mathematical_symbols_informative: Ward cannot contain mathematical symbols
+            contains_restricted_characters_instructional: Ward can only contain letters, numbers,
               local characters, and special characters
-            contains_too_many_words: Ward cannot have more than %{word_count}
-              words
-            contains_html_tags: Ward cannot contain HTML tags.
-            contains_url: Ward cannot contain URLs.
-            unknown_for_city: Enter a valid ward name for %{city}.
-            unknown_for_zip: Enter a valid ward name for %{zip}.
-            unknown_for_address: Ward may be incorrect.
+            contains_restricted_characters_informative: War can't contain restricted characters
+            contains_too_many_words_instructional: Ward must have fewer than %{word_count} words
+            contains_too_many_words_informative: Ward must have fewer than %{word_count} words
+            contains_html_tags_instructional: Ward can't contain HTML
+            contains_html_tags_informative: Ward can't contain HTML
+            contains_url_instructional: Ward can't contain a URL
+            contains_url_informative: Ward can't contain a URL
+            unknown_for_city_instructional: Select a valid ward for %{city}
+            unknown_for_city_informative: Ward does not match %{city}
```

Comment:
these unknowns in the spreadsheets are "warning" messages... these are errors. I took the liberty to change the context of the message. Is this acceptable?